interfaces/Parsing_stats.atd

```diff
@@ -0,0 +1,40 @@
+(*
+   Output parsing stats for a collection of projects to json.
+   This is used by semgrep-core -parsing_stats -json
+*)
+
+type t = {
+  language: string; (* should be compatible with Lang.t *)
+  global: project_stats;
+  projects: project_stats list;
+}
```

Comment:
It's usually best to default list fields to the empty list when it makes sense. Here, it would be:
```
  ~projects: project_stats list;
```
`~` means the field is optional and there's a default value. This type (a list) has an implicit default value so there's no need to provide it.

Reply:
But here we use this .atd for output information, so for sure the projects list should never be empty.